src/views/courseDetail.pug

```diff
@@ -137,6 +137,106 @@ block content
+                    |  #{t('course.student')}
+                  .px-2: small
+                    i.fa.fa-user
+                    |  #{t('course.begin')}
+                  .px-2: small
+                    i.fa.fa-clock.me-2
+                    |  #{totalHours}h
+                .image-div.text-left.mt-3
+                  img(src='/img/testimonial-2.jpg' alt='', )
+                  span
+                    b #{t('course.instructor_name')} - #{name}
+          .container-fluid.wow.fadeInUp.mt-5.tabs
+            .tab-content.mt-4
+              #Overview.tab-pane.container.active
+                h2 #{t('course.about_course')}
+                p 
+                  | #{course.description}
+                h2.mt-4
+                  | #{t('course.skill')}
+                span.badge.rounded-pill.text-white.bg-primary.px-4.py-2.m-2 Structure pages with HTML
+                span.badge.rounded-pill.text-white.bg-primary.px-4.py-2.m-2 Present data with tables
+                span.badge.rounded-pill.text-white.bg-primary.px-4.py-2.m-2 Write cleaner HTML
+              #Curriculum.container
+                h2.mt-4
+                  | #{t('course.section')}
+                  #accordionFlushExample.accordion.accordion-flush
+                    each section in sectionsWithLessons
+                      .accordion-item
+                        h2.accordion-header
+                          - var sectionId = 'flush-collapse' + section.id
+                          button.accordion-button.collapsed.fs-5(type='button' data-bs-toggle='collapse', data-bs-target=`#${sectionId}`, aria-expanded='false', aria-controls=sectionId)
+                            | #{section.name}
+                        .accordion-collapse.collapse(id=sectionId, data-bs-parent='#accordionFlushExample')
+                          .accordion-body
+                            if section.lessons.length > 0
+                              ul
+                                each lesson in section.lessons
+                                  li
+                                    .custom-lesson.py-2
+                                      i.fa.fa-video.text-danger
+                                      |   #{lesson.name} - #{lesson.time} #{t('course.hours')} 
+                            else
+                              .custom-lesson.py-2 #{t('course.course_lesson_error')} 
+              #Instructor.container
+                h2.mt-4 #{t('course.course_professor')}
+                .image-div.text-left.mt-4
+                  .row
+                    .col-lg-3.col-md-6
+                      img(src='/img/testimonial-2.jpg' alt='')
+                    .col-lg-9.col-md-6.mt-2
+                      h5 #{name}
+                      p #{t('course.professor_job')}
+                      .row
+                        .col-6
+                          p
+                            i.fa.fa-star
+                              |   4.87 #{t('course.professor_rating')}
```

Comment:
`4.87` -> giá trị này đc tính toán chớ e nhỉ?

Reply:
đoạn code này bọn em sẽ làm trong pr của professor